src/test/regress/expected/isolation_append_copy_vs_all.out

```diff
@@ -60,8 +60,9 @@ step s1-initialize: COPY append_copy FROM PROGRAM 'echo 0, a, 0 && echo 1, b, 1
 step s1-begin: BEGIN;
 step s1-copy: COPY append_copy FROM PROGRAM 'echo 5, f, 5 && echo 6, g, 6 && echo 7, h, 7 && echo 8, i, 8 && echo 9, j, 9' WITH CSV;
 step s2-task-tracker-select:
-	SET citus.task_executor_type TO "task-tracker";
-	SELECT * FROM append_copy AS t1 JOIN append_copy AS t2 ON t1.id = t2.int_data ORDER BY 1, 2, 3, 4;
+ SET citus.task_executor_type TO "task-tracker";
```

Comment:
this seems like a noop, would be nice to remove them.

Reply:
does it make sense to remove all task_executor_type statements in the tests including:
set citus.task_executor_type to 'adaptive';
reset citus.task_executor_type;

Reply:
18f7f424ff2d27c69da65f80404cfe7b291e9014 I did that, I can change it if there is any objection